client/scripts/views/pages/view_proposal/index.ts

```diff
@@ -344,6 +350,16 @@ const ProposalComment: m.Component<{
               trigger: m(Icon, { name: Icons.CHEVRON_DOWN })
             })
           ],
+          m(PopoverMenu, {
+            closeOnContentClick: true,
+            content: [
+              m(SocialSharingCarat, {
```

Comment:
Why is the `SocialSharingCarat` PopoverMenu being wrapped in another PopoverMenu here?

Reply:
Can't it just invoked like the first one?